### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by stevefaulkner**
added img no alt

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/html-aria/pull/164.html" title="Last updated on Sep 28, 2019, 2:53 PM UTC (6ac749b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/html-aria/164/e96f562...6ac749b.html" title="Last updated on Sep 28, 2019, 2:53 PM UTC (6ac749b)">Diff</a>

### CODE REVIEW

**Review by scottaohara — APPROVED** [review]
LGTM regarding content.  not sure what the travis failure is?

File: index.html [hunk 1]
```diff
@@ -1824,7 +1844,7 @@ <h2 id="docconformance">
           </tr>
           <tr id="progress" tabindex="-1">
             <td>
-              <code><a data-xref-for="HTMLElement">progress</a></code>
+              <code><a>progress</a></code>
```
Comment (marcoscaceres) [inline index.html]: If this is a  reference to the `<progress>` element, may I suggest: 

```suggestion
              [^progress^]
```

**Comment by marcoscaceres** [comment]
@scottaohara, the error is:
```
page was found but not the anchor from output.html to https://html.spec.whatwg.org/multipage/embedded-content.html#unknown-images
```

So something legit seems to be wrong. The actual page it's supposed to link to is:
```
https://html.spec.whatwg.org/multipage/images.html#unknown-images
```

We should get the actual "unknown images" thing `data-export`ed from HTML tho.

File: index.html [hunk 2]
```diff
@@ -925,6 +925,26 @@ <h2 id="docconformance">
               </p>
             </td>
           </tr>
+          <tr id="img-no-alt" tabindex="-1">
+            <td>
+              <code><a>img</a> with no <a data-cite=
+              "html/embedded-content.html#unknown-images">alt</a>=""</code>
```
Comment (marcoscaceres) [inline index.html]: This doesn't look right... "alt" should link to the attribute... but if you want to link to the concept of "Images whose contents are not known", then that should be done properly (i.e., see my other comment about getting that exported from HTML).

**Pull request merged**